_dashboards/maps.md

```diff
@@ -0,0 +1,154 @@
+
+For large datasets, you may want to avoid loading data for the whole map. To load data only for the part of the map that is currently visible, select the **Only request data around map extent** checkbox.
+{: .tip}
+
+## Using tooltips to visualize additional data
+
+Document layers show geopoint and geoshape document fields as locations on the map. To add more information to the locations, you can use tooltips. For example, you may want to to show flight delay, destination weather, and destination country in the Flight destination layer. Perform the following steps to configure tooltips to show additional data:
+
+1. In the **Layers** panel, select **Flight destination**.
+1. Select **Tooltips**. 
+1. Select the **Show tooltips** checkbox.
+1. In the **Tooltip fields** dropdown list, select the fields that you'd like to display. In this example, select `FlightDelay`, `DestWeather`, and `DestCountry`.
+1. Select the **Update** button.
+
+To view tooltips, hover over the geographical point you're interested in. One tooltip can display many data points. For example, in the Flight destination layer there are multiple flights for a single destination city. To paginate over the flights, select the city you're interested in and use the arrows in the tooltip:
+
+<img src="{{site.url}}{{site.baseurl}}/images/maps/tooltip.png" alt="Flight destination tooltip" width="450">
+
+If a point on the map contains data from multiple layers, one tooltip can display data from multiple layers. To choose a particular layer you're interested in, select the layer name in the tooltip layer selection panel. To see all layers, select **All layers**:
+
+<img src="{{site.url}}{{site.baseurl}}/images/maps/layer-selection-panel.png" alt="Tooltip with a layer selection panel" width="450">
+
+## Reordering, hiding, and deleting layers
+
+The **Layers** panel lets you reorder, hide, and delete layers:
+
+- Layers on a map are stacked on top of each other. To reorder layers, use the handle bar (two horizontal lines) icon next to the layer name to drag the layer to the desired position. 
+
+- If you'd like to hide a layer, select the show/hide (eye) icon next to the layer name. Toggle the show/hide icon to show the layer again.
+
+- To delete a layer, select the delete (trash can) icon next to the layer name.
+
+## Refreshing data for a real-time dataset
+
+If you want to visualize a real-time dataset, after adding layers to the map use the following steps to set the refresh interval:
+
+1. Select the calendar icon in the upper-right corner.
+1. Under **Refresh every**, select or enter the refresh interval (for example, 1 second).
+1. Select the **Start** button.
+
+<img src="{{site.url}}{{site.baseurl}}/images/maps/refresh.png" alt="Refreshing a map" width="450">
+
+## Saving a map
+
+To save a map with all layers that you set up, perform the following steps:
+
+1. Select the **Save** button in the upper-right corner. This brings up the **Save map** dialog.
```

Comment:
Let's not use the phrasal verb "brings up". How about something like "The **Save map** dialog appears."?

Reply:
Reworded.